<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-CAUKBV7" name="Test2" tests="1" failures="1" timestamp="17 Oct 2021 13:04:07 GMT" time="63.056" errors="0">
  <testcase name="CreateCustomer" time="56.462" classname="com.Repairshop.tests.LoginTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.Repairshop.tests.LoginTest.CreateCustomer(LoginTest.java:36)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- CreateCustomer -->
</testsuite> <!-- Test2 -->
